index.php

```diff
@@ -37,8 +37,24 @@
 
     <p><b>Latest MacPorts <a href="install.php">release</a>: <?php print $macports_version_latest; ?></b></p>
 
+    <h3 class="subhdr">MacPorts Meeting 2019</h3>
+
+    <div class="row">
+      <p>We are organising 3rd international MacPorts Meeting in Slovenia from 11-16 October. Interested developers and users can follow our <a href="<?php print $trac_url . 'wiki/Meetings/MacPortsMeeting2019'; ?>">wiki page</a> for more information.</p>
+    </div>
+
     <h3 class="subhdr">Getting involved: Students</h3>
 
+    <div class="row">
+        <div class="column">
+          <p>A good way for students to get involved is through <a href="https://summerofcode.withgoogle.com">GSoC</a>. We shall participate next year as well. You may find <a href="<?php print $trac_url . 'wiki/SummerOfCodeArchive'; ?>">past GSoC projects here</a>. </p>
+        </div>
+        <div class="column">
+          <a href="<?php print $trac_url . 'wiki/SummerOfCode'; ?>"> <img src="img/gsoc.png"> </a>
+        </div>
+    </div>
+
+<!-- Comment GSoC code for now
     <div class="row">
         <div class="column">
             <p> <b>Google Summer of Code</b> is a program to encourage students' participation in Open Source development and offers a stipend to work on the project with an organization for three months. MacPorts has been participating in the program since 2007! </p>
```

Comment:
I like the idea about linking directly to past GSoC projects. However, this first paragraph is a good description of what "GSoC" actually is. I would leave that in, otherwise there is no explanation of what "GSoC" refers to.

Also, please do not remove the link to the ideas page. Providing more information to interested students is relevant all year.

Reply:
@raimue, I made the changes. Also, I added horizontal rows in between shortcuts. I feel that makes it more readable. I will withdraw that commit if it looks inappropriate.